programs/launcher/main.cpp

```diff
@@ -1044,27 +1060,49 @@ launcher_def::launch (eosd_def &instance, string &gts) {
   last_run.running_nodes.emplace_back (move(info));
 }
 
+#if 0
+void
+launcher_def::kill_instance(eosd_def, string sig_opt) {
+}
+#endif
```

Comment:
Can we just remove this?

Reply:
that's a placeholder for the new feature. I'll replace these commented out code later. Since you've already tagged, let's just leave this be and I'll update the pr with new code soon,